x/profile/keeper/msg_server_create_coordinator_test.go

```diff
@@ -24,39 +25,48 @@ func msgCreateCoordinator() types.MsgCreateCoordinator {
 
 func TestMsgCreateCoordinator(t *testing.T) {
 	var (
-		msg1 = msgCreateCoordinator()
-		msg2 = msgCreateCoordinator()
+		msg1        = msgCreateCoordinator()
+		msg2        = msgCreateCoordinator()
+		ctx, k, srv = setupMsgServer(t)
+		wCtx        = sdk.WrapSDKContext(ctx)
 	)
 	tests := []struct {
-		name string
-		msg  types.MsgCreateCoordinator
-		want uint64
-		err  error
+		name   string
+		msg    types.MsgCreateCoordinator
+		wantId uint64
+		err    error
 	}{
 		{
-			name: "valid coordinator 1",
-			msg:  msg1,
-			want: 0,
+			name:   "valid coordinator 1",
+			msg:    msg1,
+			wantId: 0,
 		}, {
-			name: "valid coordinator 2",
-			msg:  msg2,
-			want: 1,
+			name:   "valid coordinator 2",
+			msg:    msg2,
+			wantId: 1,
 		}, {
 			name: "already exist address",
 			msg:  msg2,
 			err:  sdkerrors.Wrap(types.ErrCoordAlreadyExist, "coordinatorId: 1"),
 		},
 	}
-	srv, ctx := setupMsgServer(t)
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := srv.CreateCoordinator(ctx, &tt.msg)
+			_, err := srv.CreateCoordinator(wCtx, &tt.msg)
```

Comment:
Why removing the check with the ID return as a response?

This is actually relevant to check to response is correct

Reply:
because we already assert the coordinator id from the keeper. but we can keep both

Reply:
When a message response is not empty we should perform tests on it

Reply:
done!!